css/dashboard.css

```diff
@@ -1,3 +1,8 @@
+div.civiremote-funding-dashboard-elements {
```

Comment:
Is the `div` element selector really necessary?

Reply:
No, changed to select the class only.